src/App.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './App.scss';
 
+import { Person } from './components/Person/Person';
```

Comment:
no need to create a separate folder (Person) for a single file. A separate folder is required if the component (parent component) uses other (child) components related only to this parent component. But in this case `index.js` file should be added in the folder to re-export parent component

Reply:
Demo link fixed.
I don't quite understand the comment about a separate folder for the person file. The person folder was already in the pool request. Do I need to put the person file in the component folder? like this "./components/Person"?